tests/software_tests/diagnostic_configuration/test_ecu_configuration.py

```
@@ -0,0 +1,806 @@
+        self._patcher_frozenset.stop()
+        self._patcher_mapping_proxy_type.stop()
+        self._patcher_getitem.stop()
+        self._patcher_validate_raw_byte.stop()
+        self._patcher_validate_raw_2byte_value.stop()
+        self._patcher_is_request_sid.stop()
+        self._patcher_is_response_sid.stop()
+        self._patcher_translator_decode.stop()
+
+    # __init__
+
+    @pytest.mark.parametrize("states, sid_restrictions, subfunction_restrictions, did_restrictions, rid_restrictions", [
```

Ruff (0.15.20)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)